src/main/groovy/edu/oregonstate/mist/invencycle/core/Bike.groovy

```diff
@@ -0,0 +1,42 @@
+package edu.oregonstate.mist.invencycle.core
+
+class Bike {
+    Integer id
+    String make
+    String model
+    Integer year
+    String type
+    String frame_size_name
+    Integer frame_size_cm
+    Integer msrp
+    String derailuer_make_front
+    String derailuer_model_front
+    Integer derailuer_speeds_front
+    String derailuer_make_rear
+    String derailuer_model_rear
+    Integer derailuer_speeds_rear
+    String fork_make
+    String fork_model
+    Integer fork_travel
+    String shock_make
+    String shock_model
+    Integer shock_travel
+    String rim_make_front
+    String rim_model_front
+    Integer wheel_size_front
+    String hub_make_front
+    String hub_model_front
+    String rim_make_rear
+    String rim_model_rear
+    Integer wheel_size_rear
+    String hub_make_rear
+    String hub_model_rear
+    String brake_make_front
+    String brake_model_front
+    Integer brake_rotor_size_front
+    String brake_make_rear
+    String brake_model_rear
+    Integer brake_rotor_size_rear
```

Comment:
Let's remove the extra whitespace at the end of the class since they don't aid in readability. Codenarc should have complained about this. Also, maybe add new lines in between the fields to group fields that are related.